src/test/java/io/github/hildi/can/service/StandardJavaSerializationServiceTest.java

```diff
@@ -0,0 +1,71 @@
+package io.github.hildi.can.service;
+
+import io.github.hildi.can.exceptions.DeserializeException;
+import io.github.hildi.can.exceptions.FileIsNotExistsException;
+import io.github.hildi.can.exceptions.NotWritableFileException;
+import io.github.hildi.can.model.User;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+
+import java.io.File;
+import java.io.IOException;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.Mockito.mock;
+
+/**
+ * Created by Serhii Hildi on 15.03.19.
+ */
+class StandardJavaSerializationServiceTest {
+
+    private File file;
+    private StandardJavaSerializationService service;
+    private User user;
+    private File mock;
+
+    @BeforeEach
+    void setUp() throws IOException {
+        service = new StandardJavaSerializationService();
+        file = File.createTempFile("temp","txt");
+        mock = mock(File.class);
+        user = new User(1L, "Lord");
+    }
+
+    @Test
+    void shouldReturnTrueIfSerializeDataEqualsDeserializeData() {
+        service.serialize(user, file);
+        User deserialize = service.deserialize(file);
+
+        User expected = user;
+
+        assertEquals(expected, deserialize);
+    }
+
+    @Test
+    void shouldThrowDeserializeException() {
```

Comment:
again, why?
when you name something `should throw` please describe under what conditions.
the good example is the test below -  `shouldThrowFileIsNotExistsExceptionIfFileIsNotExists`.
from that test name, it is clear that some exception must be thrown when the given file doesn't exist.